README.md

```diff
@@ -97,6 +113,16 @@ The files and directories currently copied (as long as they exist in the source
 - `snippets` - CSS snippets for customizing Obsidian's appearance
 - `README.md` - optional file used for vaults distributed to others via Git
 
+See the `ITEMS_TO_COPY` global in the code to configure this.
```

Comment:
Yeah, I wish this were easier to keep in sync.
What I have done in other work projects is tell folks to run the program with a `--show-items` command, which is like `--list` here, but for ITEMS_TO_COPY instead of the vault list.
I prefer self-documenting code as much as possible, and self-describing code, but I see the value of having this written down here for folks kicking the tires.

Reply:
Of course that is moot if the list is lifted out of the code. Just thinking out loud.